### PR TITLE
Fixed various components of the outdated Swfit and amplify sdk

### PR DESCRIPTION
**Pull request by ldomb**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tutorial/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Before beginning, you must:
 
 ## Download the Source Code
 
-Download the [source code](https://github.com/aws-samples/aws-mobile-ios-notes-tutorial/archive/master.zip) from the GitHub repository.
+Download the [source code](https://github.com/ldomb/aws-mobile-ios-notes-tutorial/archive/master.zip) from the GitHub repository.
```
Comment (onlybakam) [inline tutorial/index.md]: should point to `aws-samples` or make the link relative

Comment (ldomb) [reply tutorial/index.md]: fixed it in the new pull request

**Comment by ldomb** [comment]
fixed url

**Review by onlybakam — APPROVED** [review]
looks good

**Pull request merged**